src/assets/base.css

```diff
@@ -73,5 +73,5 @@ html, body {
   position: relative;
   transition: opacity 0.5s;
   margin: 0;
-  height: 100%;
+  height: 105%;
```

Comment:
Are you sure we need this? I see that it moves the map down (which can be useful), but we need to decide as a team whether we want to solve the issue this way or by moving the nav bar.